dev/ComboBox/ComboBox_themeresources.xaml

```diff
@@ -347,8 +347,8 @@
             <Thickness x:Key="ComboBoxDropdownContentMargin">0,4,0,4</Thickness>
             <StaticResource x:Key="ComboBoxDropDownBackgroundPointerOver" ResourceKey="SystemControlBackgroundListLowBrush" />
             <StaticResource x:Key="ComboBoxDropDownBackgroundPointerPressed" ResourceKey="SystemControlBackgroundListMediumBrush" />
-            <StaticResource x:Key="ComboBoxFocusedDropDownBackgroundPointerOver" ResourceKey="SystemControlBackgroundChromeBlackLowBrush" />
-            <StaticResource x:Key="ComboBoxFocusedDropDownBackgroundPointerPressed" ResourceKey="SystemControlBackgroundChromeBlackMediumLowBrush" />
+            <StaticResource x:Key="ComboBoxFocusedDropDownBackgroundPointerOver" ResourceKey="SystemControlBackgroundBaseMediumLowBrush" />
```

Comment:
Might be better to use SystemControlBackgroundBaseLowBrush to match the original brush and also to differentiate from the pressed state.

Reply:
Fixed